output/csv/config.go

```diff
@@ -59,7 +60,7 @@ func (c Config) Apply(cfg Config) Config {
 }
 
 // ParseArg takes an arg string and converts it to a config
-func ParseArg(arg string) (Config, error) {
+func ParseArg(arg string, logger *logrus.Logger) (Config, error) {
```

Comment:
Do you have any thoughts on how we could avoid such a breaking change to the package's pubic API? Other options I was considering was:

1. Use `Config` to indicate a soon to be deprecated argument was used, and write the log message further up the call stack where the logger is initialised (in `output.go`)
2. Move the checking of the soon to be deprecated arguments to `output.go`

Reply:
Hi, thanks for the PR!

Hhmm yeah, this is not ideal, but I think I prefer leaving it as is more than the alternatives, as currently it's clear that the warning happens where the configuration is assembled, and doing that validation outside of `config.go` would muddy things a bit.

Other than that a package like [`warnings`](https://pkg.go.dev/gopkg.in/warnings.v0) might help, though it's not very idiomatic and we wouldn't want to add another dependency because of this.

So I'd say to leave it as is, unless someone from @grafana/k6-devs has a better idea.

Reply:
Okay, thanks @imiric 

> Hhmm yeah, this is not ideal, but I think I prefer leaving it as is more than the alternatives, as currently it's clear that the warning happens where the configuration is assembled, and doing that validation outside of config.go would muddy things a bit.

Yeah, I agree - this way keeps the logic local. Interested to see what @grafana/k6-devs think.

Reply:
Perhaps, it makes sense to add this case (logging during config parsing) to #883 as "nice-to-have" to figure out how best to approach this if it comes up again?

Reply:
@yorugac Good idea, I've just added this as a nice to have to that issue. :+1: